libs/partners/mongodb/README.md

```diff
@@ -6,7 +6,7 @@ pip install -U langchain-mongodb
 ```
 
 # Usage
-- See [integrations doc](../../../docs/docs/integrations/vectorstores/mongodb.ipynb) for more in-depth usage instructions.
+- See [integrations doc](../../../docs/docs/integrations/providers/mongodb_atlas.mdx) for more in-depth usage instructions.
```

Comment:
This appears to be a mistaken suffix. Should be 
`../../../docs/docs/integrations/providers/mongodb_atlas.ipynb`